glossary/voting.md

```diff
@@ -11,3 +11,13 @@ members of the [Galactic Senate](../senate), through an [Azimuth](../azimuth)
 smart contract. Galaxies collectively make decisions about the governance of
 Azimuth and the [Arvo](../arvo) network by voting. There are two types of proposals that can be voted on:
 an [upgrade proposal](../upgrade) and a [document proposal](../docvote).
+
+A proposal is always a Yes/No vote. The vote concludes at the end of 30 days,
+or whenever an absolute majority is reached, whichever comes first. At the end
+of 30 days the option that has received the most votes is chosen.
```

Comment:
While we're at it, might be worth noting there's a quorum of a quarter of the senate. That is, at least 64 galaxies must have voted for a "yes" result to be valid.